apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -111,24 +109,27 @@ contract DisputableVoting is DisputableAragonApp, IForwarder {
     }
 
     /**
-    * @notice Initialize DisputableVoting app with `_token.symbol(): string` for governance, minimum support of `@formatPct(_supportRequiredPct)`%, minimum acceptance quorum of `@formatPct(_minAcceptQuorumPct)`%, and a voting duration of `@transformTime(_voteTime)`
+    * @notice Initialize DisputableVoting app with `_token.symbol(): string` for governance, minimum support of `@formatPct(_supportRequiredPct)`%, minimum acceptance quorum of `@formatPct(_minAcceptQuorumPct)`%, a voting duration of `@transformTime(_voteTime)`, and an overrule window of `@transformTime(_overruleWindow)`
     * @param _token MiniMeToken Address that will be used as governance token
     * @param _supportRequiredPct Percentage of yeas in cast votes for a vote to succeed (expressed as a percentage of 10^18; eg. 10^16 = 1%, 10^18 = 100%)
     * @param _minAcceptQuorumPct Percentage of yeas in total possible votes for a vote to succeed (expressed as a percentage of 10^18; eg. 10^16 = 1%, 10^18 = 100%)
-    * @param _voteTime Seconds that a vote will be open for tokenholders to vote
+    * @param _voteTime Seconds that a vote will be open for token holders to vote
+    * @param _overruleWindow New overrule window in seconds
     */
-    function initialize(MiniMeToken _token, uint64 _supportRequiredPct, uint64 _minAcceptQuorumPct, uint64 _voteTime) external onlyInit {
+    function initialize(MiniMeToken _token, uint64 _supportRequiredPct, uint64 _minAcceptQuorumPct, uint64 _voteTime, uint64 _overruleWindow)
```

Comment:
Should we keep the `onlyInit` modifier?

Reply:
it is already ensured by `initialized()`, do you prefer duplicating it?

Reply:
🤦‍♀️ good point, we should remove our rule of placing the modifier here then (not sure if it's in the docs, but it's almost everywhere we use `initialize()`).